### ISSUE TITLE
It's not clear to me how to open sources so I can debug

### ISSUE THREAD

**Issue by TCourtneyOwen** [microsoft/vscode-edge-devtools#205]
I have opened a webpage in Edge tools pane and see this pane, but don't see a way to open sources so I can set breakpoints.  I want to debug taskpane.js.

![image](https://user-images.githubusercontent.com/38081557/94947257-32443800-0492-11eb-96ae-8d33db678449.png)

Am I missing something here? 

Thanks,

Courtney

**Comment by vidorteg** [comment]
Hi @TCourtneyOwen,

Thanks for trying the extension. I'd separate this into two different issues:

- **Cannot set breakpoints:** I recommend The [Debugger for Microsoft Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) extension for this, it complements perfectly with the Microsoft Edge Tools For VS Code perfectly as you can use one for debugging your scripts and the other one for debugging your UI.

- **Cannot see sources:** We focused our initial efforts in the "Elements" and "Network tab", but we would love your feedback around which tools or features would you like to see next. For the specific case of the sources panel, you can get a similar behavior using the "Debugger for Microsoft Edge" extension.

Does this helps?

**Comment by keckelt** [comment]
Hi @vidorteg,

regarding:
> Cannot set breakpoints: I recommend The Debugger for Microsoft Edge extension for this, it complements perfectly with the Microsoft Edge Tools For VS Code as you can use one for debugging your scripts and the other one for debugging your UI.

Can both be done in parallel? I.e. step through the code with the Debugger and see DevTools?

**Comment by darthwalsh** [comment]
@vidorteg can the team show an example of using both the devtools and the debugger extension together in launch.json? I don't want to become a whizz at composing vs code extensions just to use vs code as my single tool to debug JS and DOM.

**Comment by darthwalsh** [comment]
Oh, now I found the docs which say the devtools should automagically attach when you use the debugger extension: https://github.com/microsoft/vscode-edge-devtools#attaching-automatically-when-launching-the-browser-for-debugging but it's not working for me, and I can't find any logs in OUTPUT related to either Edge Devtools or Edge Debugger as to why it didn't work.

I also wasn't able to get both attached by launching the Debugger then manually attaching the Devtools, so maybe that's why the automagic didn't work. Some more docs for how to debug when automagic doesn't work would be helpful to me :)

I was able to get both attached by launching Devtools first, then manually attaching the Debugger.
